cmd/kots/cli/install.go

```diff
@@ -643,3 +656,81 @@ func getHttpProxyEnv(v *viper.Viper) map[string]string {
 	return env
 
 }
+
+func callback(t uint32, f func() bool) func() {
+	return func() {
+		if f() {
+			time.AfterFunc(time.Duration(t)*time.Second, callback(t, f))
+		}
+	}
+}
+
+//IsAppReady checks for app readiness
+func IsAppReady(namespace string) (bool, error) {
+
+	log := logger.NewLogger()
```

Comment:
remove

Reply:
logger is needed to pass into the PortForward function. its ok to keep it,